types/react-native/index.d.ts

```diff
@@ -500,6 +520,16 @@ export interface PressableProps extends AccessibilityProps, Omit<ViewProps, 'sty
      */
     cancelable?: null | boolean | undefined;
 
+    /**
+     * Duration to wait after hover in before calling `onHoverIn`.
+     */
+    delayHoverIn?: number,
```

Comment:
As this is a [maybe type](https://flow.org/en/docs/types/maybe/) according to the [source code](https://github.com/facebook/react-native/blob/main/Libraries/Pressability/Pressability.js#L58), I think it should be reflected here as 

```suggestion
    delayHoverIn?: number | null | undefined,
```

Same below :)

Reply:
Sorry, missed this and just fixed it. Thanks for the catch!